nixos/modules/services/misc/navidrome.nix

```diff
@@ -0,0 +1,116 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.navidrome;
+  configFile = "/etc/navidrome/settings.json";
+  dataFolder = "/var/lib/navidrome";
+in {
+  options = {
+
+    services.navidrome = {
+      enable = mkEnableOption "Enable Navidrome music server and streamer";
+
+      settings = lib.mkOption {
+        type = types.attrs;
+        default = {};
+        example = {
```

Comment:
You should use `literalExample` to render this correctly in the manual, otherwise al the formatting will go away.
Use it like this:
```
literalExample ''
  Nix code here
'';
```

In general, you should build the manual and open it to check the result. 
You can build it with 
```
$ nix build -f nixos/release.nix manual.x86_64-linux
```
then open `result/share/doc/nixos/index.html` with a browser.
To check the description of your new NixOS options click "next" at the top of the page and use the find-in-page feature of the browser.

Reply:
Didn't know, now I did as you say. I also tried to build the manual and it looks nice.